ail_vru_uc1_scenario/cdasim_config/.env

```diff
@@ -1,3 +1,9 @@
 INFRASTRUCTURE_ID=rsu_1234
 SIMULATION_MODE=true
-TIME_SYNC_TOPIC=time_sync
\ No newline at end of file
+TIME_SYNC_TOPIC=time_sync
+# Left turn to north scenario
+VEHICLE_SELECTED_ROUTE=vru_uc1_north
+VEHICLE_SPAWN_POINT="215.12,-169,0,0,0,0"
+# Through to south scenario
+# VEHICLE_SELECTED_ROUTE=vru_uc1_south
```

Comment:
I know config folder have a south route, but the scenarios under test are not using them anymore. Both scenarios are vru_uc1_north now in terms of route.

Reply:
I am not sure I understand. Also I think if we can get the route and the scenario to share a name we could use a single environment variable to set both. Please provide both route names if what I have here should be changed.

Reply:
Same scenario can be reused with a different route. So it's better if they have a different name. My suggested change is line7-8:
```
# Through to north scenario
# VEHICLE_SELECTED_ROUTE=vru_uc1_north
```

Reply:
Talked offline. It is confusing because we are using same route, so we don't need to distinguish. We can delete the south route and only distinguish between spawn points (we should even remove route option from the env file). So the file would have 
```
# Left turn to north scenario spawn point
VEHICLE_SPAWN_POINT="215.12,-169,0,0,0,0"
# Through movement from south to north scenario spawn point
# VEHICLE_SPAWN_POINT="258.12,-125,0,0,0,90"
```